test/upgrade/prober/wathola/event/services.go

```diff
@@ -135,12 +175,18 @@ func (s *stepStore) reportProgress() {
 	}
 }
 
-func (e *ErrorStore) throw(format string, args ...interface{}) {
+func (e *ErrorStore) throw(tag string, format string, args ...interface{}) {
```

Comment:
Instead of adding a `tag string` just implement different methods. The added if inside is just a bad, unclean, wobbly code. Let's avoid that.

Reply:
Make sense, use method instead of string.